package.json

```diff
@@ -127,10 +129,17 @@
   },
   "scripts": {
     "bootstrap": "npm install && cd addon && npm install && cd .. && gulp",
-    "start": "cross-env USE_HTTPS=1 NODE_ENV=development ENABLE_DEV_LOCALES=1 ENABLE_DEV_CONTENT=1 gulp",
+    "start": "cross-env NODE_ENV=development ENABLE_DEV_LOCALES=1 ENABLE_DEV_CONTENT=1 npm-run-all build:gulp watch",
```

Comment:
Having split webpack out into its own build step complicated things a tiny bit: Gulp having built `experiments.json` is a prerequisite for webpack, because we `require()` it from the build directory in `frontend/src/store.js`